src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage.Accessor.cs

```diff
@@ -158,52 +146,54 @@ public Task<Stream> ReadStreamAsync(TKey key, Checksum checksum, CancellationTok
                 return false;
             }
 
-            private Task FlushPendingWritesAsync(TKey key, CancellationToken cancellationToken)
-                => Storage.FlushSpecificWritesAsync(_writeQueueKeyToWrites, _writeQueueKeyToWriteTask, GetWriteQueueKey(key), cancellationToken);
-
-            private Task AddWriteTaskAsync(TKey key, Action<SqlConnection> action, CancellationToken cancellationToken)
-                => Storage.AddWriteTaskAsync(_writeQueueKeyToWrites, GetWriteQueueKey(key), action, cancellationToken);
-
             private Stream ReadBlob(
-                SqlConnection connection, TDatabaseId dataId, string columnName,
+                SqlConnection connection, string dbName, TDatabaseId dataId, string columnName,
                 Checksum checksumOpt, CancellationToken cancellationToken)
             {
-                // Note: it's possible that someone may write to this row between when we
-                // get the row ID above and now.  That's fine.  We'll just read the new
-                // bytes that have been written to this location.  Note that only the
-                // data for a row in our system can change, the ID will always stay the
-                // same, and the data will always be valid for our ID.  So there is no
-                // safety issue here.
-                if (TryGetRowId(connection, dataId, out var rowId))
+                try
```

Comment:
i moved this try/catch from teh caller here, since we now call this twice with two different dbNames.  This prevents havingt he caller have to have the dual try/catches